packages/prepare-commit-msg/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@coauthors/prepare-commit-msg",
   "version": "0.0.2",
-  "private": true,
```

Comment:
This change will make @coauthors/prepare-commit-msg be deployed by changeset's release PR